Sources/Internal/VisibleItemsProvider.swift

```diff
@@ -614,7 +618,7 @@ final class VisibleItemsProvider {
 
         let itemType = VisibleCalendarItem.ItemType.layoutItemType(layoutItem.itemType)
 
-        let calendarItemModel: InternalAnyCalendarItemModel
+        var calendarItemModel: InternalAnyCalendarItemModel? = nil
```

Comment:
```suggestion
        let calendarItemModel: InternalAnyCalendarItemModel?
```

Let's keep this as a `let` so that it's clear that it can only be set once.